cluster.go

```diff
@@ -761,6 +762,7 @@ This might be due to one or several causes:
 	c.shutdownLock.Lock()
 	c.readyB = true
 	c.shutdownLock.Unlock()
+	daemon.SdNotify(false, daemon.SdNotifyReady)
```

Comment:
Hmm I don't think this is the best place, as cluster is supposed to work as a library included in some other daemon.

I think you should launch a goroutine that waits for "Ready()" here: https://github.com/ipfs-cluster/ipfs-cluster/blob/master/cmd/ipfs-cluster-service/daemon.go#L107

and do the notification then.

Reply:
I turned it into a goroutine as you suggested. Please let me know if my implementation looks appropriate, I'm not particularly experienced in Go development.

Reply:
Yeah looks good to me now!